apps/maestro/src/features/InterchainTokenDeployment/InterchainTokenDeployment.state.ts

```diff
@@ -131,6 +132,12 @@ function useInterchainTokenDeploymentState(
           ...draft.tokenDetails,
           ...partialInitialState.tokenDetails,
         };
+
+        // Make sure isCanonical is properly set but not part of form validation
+        if (partialInitialState.tokenDetails?.isCanonical !== undefined) {
+          draft.tokenDetails.isCanonical =
+            partialInitialState.tokenDetails.isCanonical;
```

Comment:
shouldn't `isCanonical` always be false in this `InterchainTokenDeployment` file?